src/utils.js

```diff
@@ -588,4 +588,12 @@ export function isSrcdocSupported(doc) {
 
 export function cloneJson(obj) {
   return JSON.parse(JSON.stringify(obj));
+}
+
+export function inIframe() {
```

Comment:
I think the problem here is we need to check a passed in window param, which we don't have. I think we have to change the `renderAd` API to accept the window instead of doc :(